diff --git a/node_modules/local-devices/src/index.js b/node_modules/local-devices/src/index.js
index 66dd8b5..bfb00b5 100644
--- a/node_modules/local-devices/src/index.js
+++ b/node_modules/local-devices/src/index.js
@@ -2,7 +2,8 @@ var ip = require('ip')
 var os = require('os')
 var net = require('net')
 var cp = require('mz/child_process')
-var getIPRange = require('get-ip-range')
+var getIPRangePackage = require('get-ip-range')
+var getIPRange = getIPRangePackage.getIPRange || getIPRangePackage
 
 var parseLinux = require('./parser/linux')
 var parseWin32 = require('./parser/win32')
